### PR TITLE
games-engines/devilutionx: Change from Unlicense to Sustainable-Use-1.0

### PR DESCRIPTION
**Pull request by lanodan**
---

Please check all the boxes that apply:

- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [ ] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

Please note that all boxes must be checked for the pull request to be merged.

### CODE REVIEW

File: games-engines/devilutionx/devilutionx-1.5.3.ebuild [hunk 1]
```diff
@@ -13,7 +13,7 @@ SRC_URI="
 "
 S=${WORKDIR}/${PN}-src-${PV}
 
-LICENSE="Unlicense BSD CC-BY-4.0 GPL-2+ LGPL-2.1+ MIT OFL-1.1 zerotier? ( BUSL-1.1 )"
```
Comment (thesamesam) [inline games-engines/devilutionx/devilutionx-1.5.3.ebuild:16]: A revbump is needed as the package now becomes non-free.

**Review by ionenwks — APPROVED** [review]
Thanks, my bad for missing this change when I added 1.5.0.

Doesn't really fit any license groups at a glance, so nothing else to do I think. I'll merge in a bit.

**Comment by ionenwks** [comment]
> Thanks, my bad for missing this change when I added 1.5.0.
> 
> Doesn't really fit any license groups at a glance, so nothing else to do I think. I'll merge in a bit.

Actually wait, ... since not even binary-redist, maybe we should slap RESTRICT="bindist" -- not sure if we need mirror too.

**Comment by lanodan** [comment]
>Actually wait, ... since not even binary-redist, maybe we should slap RESTRICT="bindist" -- not sure if we need mirror too.

Yeah, looking at it again:

* RESTRICT=bindist seems unclear to me, so I guess better better
   treat it as restricted by default due to how the license is worded

* RESTRICT=mirror would make sense due the non-commercial limitation
   and anyone being able to make mirrors

**Comment by ionenwks** [comment]
Thanks, went ahead and added `bindist mirror` too.

**Pull request closed without merging**